client/javascripts/Transport.elm

```diff
@@ -0,0 +1,35 @@
+module Transport exposing (..)
+
+import Html exposing (..)
+
+-- MAIN
+
+main : Html String
+main = view
+
+-- VIEW
+
+view : Html String
+view =
+    section []
+    [ h1 [] [ text "transport.beta.gouv.fr" ]
+    , h2 [] [ text "Rendre disponible, valoriser et améliorer les données transports" ]
+    , h3 [] [ text "transport.beta.gouv.fr c'est quoi ?" ]
+    , p  [] [ text "C'est la plateforme de diffusion des données ouvertes relatives au transport. Mais c'est aussi la plateforme qui permet à la communauté des diffuseurs et des ré-utilisateurs de se rassembler, de dialoguer, de réfléchir ensemble aux problèmes autour des données transport (qualité, complétude, licences) et de construire les outils pour les résoudre." ]
+    , h3 [] [ text "Pourquoi transport.beta.gouv.fr ?" ]
```

Comment:
I might be tempted to *start* refactoring to let latent structure appear. It's a good thing not to push it too far in the early stages of an app, lest you end up with premature abstractions, but that's quite a large function. But it's hard to tell how to handle that without knowing more about where you're heading next.

Reply:
@Morendil Thing is I don't yet how much of that content will make it to the landing page. I'll greatly appreciate a text review as well. I haven't had the time to enable review apps yet, sadly, but running the app locally should be trivial if you follow the README.